lib/_tls_common.js

```diff
@@ -169,21 +169,18 @@ exports.translatePeerCertificate = function translatePeerCertificate(c) {
   if (!c)
     return null;
 
-  if (c.issuer) c.issuer = tls.parseCertString(c.issuer);
-  if (c.issuerCertificate && c.issuerCertificate !== c) {
+  if (c.issuer != null) c.issuer = tls.parseCertString(c.issuer);
+  if (c.issuerCertificate != null && c.issuerCertificate !== c) {
     c.issuerCertificate = translatePeerCertificate(c.issuerCertificate);
   }
-  if (c.subject) c.subject = tls.parseCertString(c.subject);
-  if (c.infoAccess) {
+  if (c.subject != null) c.subject = tls.parseCertString(c.subject);
+  if (c.infoAccess != null) {
     var info = c.infoAccess;
-    c.infoAccess = {};
+    c.infoAccess = Object.create(null);
 
     // XXX: More key validation?
     info.replace(/([^\n:]*):([^\n]*)(?:\n|$)/g, function(all, key, val) {
-      if (key === '__proto__')
-        return;
-
-      if (c.infoAccess.hasOwnProperty(key))
+      if (key in c.infoAccess)
```

Comment:
Unlikely to matter because it's only a handful of keys (if it's set at all.)  This is shorter so that's what I picked.